src/main/java/persistence/entity/EntityMeta.java

```diff
@@ -15,12 +15,19 @@ public EntityMeta(Table table, IdColumn id) {
         this.id = id;
     }
 
-    public static <T> EntityMeta from(T entity) {
+    public static <T> EntityMeta<T> from(T entity) {
         Table table = Table.from(entity.getClass());
         IdColumn id = table.getIdColumn();
-        return new EntityMeta(table, id);
+        return new EntityMeta<T>(table, id);
     }
 
+    public static <T> EntityMeta<T> from(Class<?> clazz) {
+        Table table = Table.from(clazz);
+        IdColumn id = table.getIdColumn();
+        return new EntityMeta<T>(table, id);
+    }
```

Comment:
```suggestion
    public static <T> EntityMeta<T> from(Class<?> clazz) {
        Table table = Table.from(clazz);
        IdColumn id = table.getIdColumn();
        return new EntityMeta<>(table, id);
    }
```
사소하지만 new 로 생성시에는 타입추론이 되어서 제거되어도 될 것 같아요 :)

Reply:
확인 감사합니다 ☺️
제거하도록 하겠습니다!ㅎㅎ